src/containers/scenes/aoi-scene/index.js

```diff
@@ -136,11 +136,20 @@ function AOIScene(props) {
 
   // Reconcile all data until completely loaded
   useEffect(() => {
+    const hasWDPAData =
+      contextualData && (contextualData.WDPAID || contextualData.DESIG_T);
```

Comment:
If we are going to introduce a condition maybe we can check with precalculatedSlug && country.iso.
If slug is wdpa but we have countryISO, this countryData should not appear.

Reply:
But maybe we have the same problem with regions or other AOI. So I think its better to know if we have the PA data